src/views/virtualmachines/migrate/VirtualMachineMigration.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  modelToGroupVersionKind,
+  StorageClassModel,
+  VirtualMachineModelGroupVersionKind,
+} from '@kubevirt-utils/models';
+import { getName } from '@kubevirt-utils/resources/shared';
+import useDisksSources from '@kubevirt-utils/resources/vm/hooks/disk/useDisksSources';
+import { ResourceLink, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  Bullseye,
+  Stack,
+  StackItem,
+  Title,
+  Wizard,
+  WizardHeader,
+  WizardStep,
+} from '@patternfly/react-core';
+import { Table, Tbody, Td, Tr } from '@patternfly/react-table';
+
+import { migrateVM } from './utils';
+
+const StorageClassModelGroupVersionKind = modelToGroupVersionKind(StorageClassModel);
+
+const VirtualMachineMigrate: FC = () => {
+  const { t } = useKubevirtTranslation();
+
+  const { name, namespace } = useParams<{ name: string; namespace: string }>();
+  const [selectedStorageClass, setSelectedStorageClass] = useState('');
+  const [migrationError, setMigrationError] = useState<Error | null>(null);
+  const [migrationLoading, setMigrationLoading] = useState(false);
+
+  const navigate = useNavigate();
+
+  const [vm, loaded, error] = useK8sWatchResource<V1VirtualMachine>(
+    name
+      ? {
+          groupVersionKind: VirtualMachineModelGroupVersionKind,
+          name,
+          namespace,
+        }
+      : null,
+  );
+
+  const { pvcs } = useDisksSources(vm);
+
+  const [{ clusterDefaultStorageClass, sortedStorageClasses }, scLoaded] = useDefaultStorageClass();
+
+  const defaultStorageClassName = getName(clusterDefaultStorageClass);
+
+  const destinationStorageClass = selectedStorageClass || defaultStorageClassName;
+
+  const onSubmit = async () => {
+    setMigrationLoading(true);
+    setMigrationError(null);
+    try {
+      await migrateVM(vm, pvcs, destinationStorageClass);
+
+      navigate(-1);
+    } catch (apiError) {
+      setMigrationError(apiError);
+    }
+
+    setMigrationLoading(false);
+  };
+
+  if (!loaded && !scLoaded)
+    return (
+      <Bullseye>
+        <Loading />
+      </Bullseye>
+    );
+
+  if (error)
+    return (
+      <Alert isInline title={t('Error')} variant={AlertVariant.danger}>
+        {error?.message}
+      </Alert>
+    );
+
+  return (
+    <Wizard
+      header={
+        <WizardHeader
+          closeButtonAriaLabel={t('Close header')}
+          description={t('Migrate VirtualMachine storage to a different StorageClass.')}
+          title={t('Migrate VirtualMachine storage')}
+        />
+      }
+      onClose={() => navigate(-1)}
+      onSave={onSubmit}
+      title={t('Migrate VirtualMachine storage')}
+    >
+      <WizardStep id="wizard-migrate-destination" name={t('Destination StorageClass')}>
+        <Stack hasGutter>
+          <StackItem>
+            <Title headingLevel="h2">{t('Destination StorageClass')}</Title>
+            <p>{t('Select the destination storage for the VirtualMachine storage migration.')}</p>
```

Comment:
You can use Text/Typography from pattternfly; it better because it aimed to font and size of p across all PF components